examples/singlethread/bqc/app_client.py

```diff
@@ -63,7 +63,7 @@ def main(
 
     with client:
         # Create EPR pair
-        epr1 = epr_socket.create()[0]
+        epr1 = epr_socket.create_keep()[0]
```

Comment:
I have a general question here: why do we need to explicitly keep the `epr_socket` here (and in the `examples/singlethread/bqc/app_server.py` file)?

Reply:
I changed the interface for the EPR Socket so that types are more clear to the programmer. Before, you had to call `create` and it could return many different types of objects (`Union[List[Qubit], List[EprMeasureResult], List[LinkLayerOKTypeM]]`) depending on which type of EPR creation you wanted (like create or keep). Now these different types are split across separate methods, for which it is clear what type they return. For example `create_keep` just returns `List[Qubit]`.